figure.html

```diff
@@ -160,13 +188,31 @@
 
 <div class="note">
 <p class="note-title">Note</p>
-<p class="note-text">If you select more than one image and they do not have the same number of channels, Z sections or time points, those controls will be greyed-out in the Preview tab.<br />
+<p class="note-text">If multi-selected images do not have the same number of channels, Z sections or time points, those controls will be greyed-out in the Preview tab.<br />
 If you select two or more images that have different settings, for example, different channels switched on and off, different zoom levels, or pan locations, then these can be resynchronised by changing the setting while they are selected.</p>
 </div><!-- End note -->
 
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
+
+<li>
+<p class="first">Panels can be zoomed, panned by clicking-and-dragging, and the image rotated.</p>
```

Comment:
Probably need to indicate that the clicking and dragging has to be in the Preview tab viewer to do panning, not the figure panel itself.